### DISCUSSION TITLE
Is there any way to change text formatting (spacing, font size etc.)?

### DISCUSSION THREAD

**Discussion by pe-hy** [executablebooks/meta#160] [Q&A]
Is there any way to configure how the final HTML and PDF output via LaTeX will look like? I want to for example change the line spacing. Haven't been able to figure this out. Thank you in advance.

**Comment by choldgraf** [comment]
Hmm - right now your best bet is to use some custom CSS to accomplish this (for the HTML or PDF via HTML outputs): https://jupyterbook.org/advanced/sphinx.html#custom-css-or-javascript
